src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java

```diff
@@ -25,6 +29,8 @@
         implements ColumnVisitor
 {
     private static final JsonParser JSON_PARSER = new JsonParser();
+    private static final List<String> BOOL_TRUE_STRINGS = ImmutableList.of("true", "1", "yes");
```

Comment:
BTW: Is it better to follow CSVParserPlugin to convert string keywords to bool? What do you think about this?